librz/core/cil.c

```diff
@@ -422,50 +422,203 @@ RZ_IPI void rz_core_analysis_rzil_reinit(RzCore *core) {
 	}
 }
 
-static void rzil_print_register(int padding, const char *reg_name, RzBitVector *number, RzStrBuf *sb) {
+static void rzil_print_register_bool(const char *reg_name, bool value, RzStrBuf *sb, RzTable *tbl, PJ *pj) {
```

Comment:
Instead of passing `sb`, `pj`, and `tbl` every time, you could create some kind of context union and use in all these functions.